Sources/Spezi/Spezi/View+Spezi.swift

```diff
@@ -26,13 +26,74 @@ struct SpeziViewModifier: ViewModifier {
 }
 
 
+/// Options to simulate behavior for a ``LifecycleHandler`` in cases where there is no app delegate like in Preview setups.
+public enum LifecycleSimulationOptions {
+    /// Simulation is disabled.
+    case disabled
+    /// The ``LifecycleHandler/willFinishLaunchingWithOptions(_:launchOptions:)-8jatp`` method will be called for all
+    /// configured ``Module``s that conform to ``LifecycleHandler``.
+    case launchWithOptions(_ launchOptions: [UIApplication.LaunchOptionsKey: Any])
+}
+
+
 extension View {
-    /// Use the `spezi()` `View` modifier to configure Spezi for your application.
-    /// - Parameter delegate: The `SpeziAppDelegate` used in the SwiftUI `App` instance.
-    /// - Returns: A SwiftUI view configured using the Spezi framework
+    /// Configure Spezi for your application using a delegate.
+    /// - Parameter delegate: The ``SpeziAppDelegate`` used in the SwiftUI App instance.
+    /// - Returns: The configured view using the Spezi framework.
     public func spezi(_ delegate: SpeziAppDelegate) -> some View {
         modifier(SpeziViewModifier(delegate.spezi))
     }
+
+    /// Configure Spezi for your previews using a Standard and a collection of Modules.
+    ///
+    /// This modifier can be used to configure Spezi with a Standard a collection of Modules without declaring a ``SpeziAppDelegate``.
+    ///
+    /// - Important: This modifier is only recommended for Previews. As it doesn't configure a ``SpeziAppDelegate`` lifecycle handling
+    ///     functionality, using ``LifecycleHandler``,  of modules won't work.
```

Comment:
The Lifecycle comment should be update to reflect the option. Should probably add an element that this should only be used in preview and crashes otherwise.